src/TestFramework/Config/TestFrameworkConfigLocator.php

```diff
@@ -27,16 +27,17 @@ public function locate(string $testFrameworkName, string $customDir = null): str
 
         foreach (['xml', 'yml'] as $extension) {
             $conf = sprintf('%s/%s.%s', $dir, $testFrameworkName, $extension);
+            $altConf = sprintf('%s/%s.dist.%s', $dir, $testFrameworkName, $extension);
 
             if (file_exists($conf)) {
                 return realpath($conf);
-            }
-
-            if (file_exists($conf . '.dist')) {
+            } elseif (file_exists($conf . '.dist')) {
                 return realpath($conf . '.dist');
+            } elseif (file_exists($altConf)) {
+                return realpath($altConf);
             }
         }
 
-        throw new \RuntimeException(sprintf('Unable to locate %s.(xml|yml)(.dist) file.', $testFrameworkName));
+        throw new \RuntimeException(sprintf('Unable to locate %s(.dist).(xml|yml)(.dist) file.', $testFrameworkName));
```

Comment:
This sentence is becoming a bit confusing, maybe we should have something like
`Unable to locate %s.(xml|yml) or dist version.` 

What do you think?

Reply:
And also can we avoid of using RuntimeException? for we already have LocatorException class where we can specify our exception. I think it makes sense

Reply:
```php
foreach (...) {
    //...
    $tried[] = $conf;
}

throw new X(sprintf('Unable to locate either of: %s', implode(', ', $tried));
```